elibrary-core/source/resources/messages.properties

```diff
@@ -184,7 +184,7 @@ loanServicePanel.loanLimitExpiredErrorTitle                                   =
 loanServicePanel.loanPanelTitle                                               = Loans
 loanServicePanel.notSelectedLoanError                                         = Loan does not have been selected. The action can not been performed.
 loanServicePanel.notSelectedLoanErrorTitle                                    = Not enough data
-loanServicePanel.penaltyError                                                 = User have to pay penalty because of untimely book returning. Penalty is:
+loanServicePanel.penaltyError=User have to pay penalty because of untimely book returning. Penalty is: {0}
```

Comment:
Please format the file using `diary/elibrary-intellij-formatter.xml`, and import the formatter in IDE configuration, for IntelliJ it's:
File > Settings > Editor > Code Style > Java and then:
![image](https://user-images.githubusercontent.com/26207880/207083936-2e8f0f4a-c9f2-476e-99db-a0ff8dacfbee.png)

Please use that formatter in the project and always ensure that before the changes are released, the format& optimize imports actions are porformed.